vendor/engines/saml_authentication/lib/saml_authentication/saml_authenticatable_with_custom_error.rb

```diff
@@ -22,11 +22,26 @@ class SamlAuthenticatableWithCustomError < SamlAuthenticatable
       # authentication to call fail!(:saml_invalid), which allows us to specify a separate
       # failure message for authentication via SAML.
       def failed_auth(msg)
+        Rollbar.warn(msg, attributes: failed_auth_user_attributes) if defined?(Rollbar)
```

Comment:
This might get noisy - what do you think @jossim ?

Reply:
@giladshanan do we know anything else about the error? If so, perhaps specific errors that are not of interest could be filtered out or something like that.

Reply:
@jossim good call